Octokit.Tests.Conventions/Exception/InterfaceNotFoundException.cs

```diff
@@ -8,18 +8,20 @@ public class InterfaceNotFoundException : Exception
         public InterfaceNotFoundException() { }
 
         public InterfaceNotFoundException(string type)
-            : base(CreateMessage(type)) { }
+            : base(CreateMessage(type))
+        { }
 
```

Comment:
aaaahhh...sorry, I must have forgotten to put on my :eyeglasses: when I looked over this...it was the indentation that got me...:nothingtoseehere: :movealong: :flushed: